tests/integrations/aiohttp/test_aiohttp.py

```diff
@@ -120,6 +123,28 @@
     assert not events
 
 
+async def test_cancelled_error_not_captured(
+    sentry_init, aiohttp_client, loop, capture_events
+):
+    sentry_init(integrations=[AioHttpIntegration()])
+
+    async def hello(request):
+        raise asyncio.CancelledError()
+
+    app = web.Application()
+    app.router.add_get("/", hello)
+
+    events = capture_events()
+    client = await aiohttp_client(app)
+
+    with suppress(ServerDisconnectedError):
```

Comment:
please use `pytest.raises` here unless you are saying this error does not happen all the time

Reply:
I've used `suppress` to avoid testing `aiohttp` here.

At the moment, `ServerDisconnectedError` is always raised, but this behavior can be changed in the future and will cause tests to fail even though `sentry-python` operates normally.

But, if you'd like to use `pytest.raises` instead, let me know :)

Reply:
ok, let's stick with this